.ci/test_r_package.sh

```diff
@@ -20,8 +20,8 @@ if [[ "${R_MAJOR_VERSION}" == "3" ]]; then
     export R_LINUX_VERSION="3.6.3-1bionic"
     export R_APT_REPO="bionic-cran35/"
 elif [[ "${R_MAJOR_VERSION}" == "4" ]]; then
-    export R_MAC_VERSION=4.1.1
-    export R_LINUX_VERSION="4.1.1-1.2004.0"
+    export R_MAC_VERSION=4.1.2
+    export R_LINUX_VERSION="4.1.2-1.2004.0"
```

Comment:
I guess we need update all R versions in this list (`4.0` -> `4.1`), not only for macOS
https://github.com/microsoft/LightGBM/blob/0075814f02f898ae4070d848078b9d0d4b510ceb/.github/workflows/r_package.yml#L30-L140

Reply:
ah yep you're right! Updated in https://github.com/microsoft/LightGBM/pull/4989/commits/7cb0fd0e4428c57c8fef7a31a263e0ff6ee921b0